src/utils/phone-number.ts

```diff
@@ -0,0 +1,14 @@
+import { isValidPhoneNumber } from "libphonenumber-js";
+
+export function containsUKMobileNumber(value: string): boolean {
+  return isValidPhoneNumber(value, "GB");
+}
+
+export function containsNumbersOrSpacesOnly(value: string): boolean {
+  return value ? /^[\d\s]+$/.test(value) : false;
+}
+
+export function lengthInRangeWithoutSpaces(value: string, min: number, max: number): boolean {
```

Comment:
can we add some unit tests around this to make sure it works as expected

Reply:
added